src/xrpld/app/main/Main.cpp

```diff
@@ -143,7 +143,7 @@ printHelp(const po::options_description& desc)
            "     connect <ip> [<port>]\n"
            "     consensus_info\n"
            "     deposit_authorized <source_account> <destination_account> "
-           "[<ledger>]\n"
+           "[<ledger> [<credentials>, ...]]\n"
```

Comment:
Shouldn't credentials come before the ledger?

Reply:
It would still be compatible with previous code, though? It would be equivalent to providing no credentials.

Reply:
No, because previous code expect ledger as 3d parameter,  not credential

Reply:
Lets look at example
`deposit_authorized  2E0B04ED60588A758B67E21FBBE95AC5A63598BA951761DC0EC9C08D7E01E034`
is this ledger or credential?
how about this one `deposit_authorized  E0B04ED60588A758B67E21FBBE95AC5A63598BA951761DC0EC9C08D7E01E034 7B77C2026AC8E674F4AEF0E720F209B298AAF7A4920C0BFB67027D9FE379814`
is this ledger, credential / credential, ledger / credential, credential ?

Reply:
Bump

Reply:
>How could you submit a request for just credentials and no ledger?
No good way, always put  "`validated`" if you want to add credentials

Reply:
Personally I'd be fine with a breaking change here, because that doesn't seem like a great API. @gregtatcam

Reply:
This parameters doesn't have names, so any other "fixes" will be like a crunch.  Or we need to change whole system and add parameter names.